### PR TITLE
Add npm eslint config package

### PR DESCRIPTION
**Pull request by alex35mil**
@justin808 @robwise Please, review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ MyComponent/
   |-- MyComponent.spec.jsx
 ```
 
-When we'll apply airbnb rule:
+With AirBnb rule applied, it looks kinda ugly and not consistent:
```
Comment (justin808) [inline README.md]: With AirBnb rule applied, it looks inconsistent.

File: packages/eslint-config-shakacode/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# eslint-config-shakacode
+
+Shakacode `eslint` config.
+
+## Install
+
+```bash
+npm install --save-dev eslint-config-shakacode eslint-config-airbnb eslint
```
Comment (justin808) [inline packages/eslint-config-shakacode/README.md:8]: :+1:

File: packages/eslint-config-shakacode/scripts/release [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```
Comment (justin808) [inline packages/eslint-config-shakacode/scripts/release:1]: fwiw, I use npm release-it

Comment (alex35mil) [inline packages/eslint-config-shakacode/scripts/release:1]: Yep, I checked it out and it has bunch of settings to learn. Most of the time I need to just release patch or minor and most of the time this little shell script do what I need, so not sure it's worth it.

File: README.md (hunk 1 above)
Comment (alex35mil) [inline README.md]: :+1:

**Pull request merged**